vnext/Microsoft.ReactNative/ReactInstanceSettings.idl

```diff
@@ -12,129 +12,199 @@ import "RedBoxHandler.idl";
+  [default_interface]
+  DOC_STRING("The arguments for the @ReactInstanceSettings.InstanceDestroyed event.")
+  runtimeclass InstanceDestroyedEventArgs
+  {
+    DOC_STRING("Gets @IReactContext for the React instance that just destroyed.")
     IReactContext Context { get; };
   }
 
   [webhosthidden]
-  DOC_STRING("Provides configuration of the react instance.")
-  runtimeclass ReactInstanceSettings 
+  DOC_STRING("Provides settings to create React instance.")
+  runtimeclass ReactInstanceSettings
   {
+    DOC_STRING("Creates new instance of @ReactInstanceSettings")
     ReactInstanceSettings();
 
-    DOC_STRING("Properties are shared with @IReactContext.Properties. It can be used to configure and share values and state between components.")
+    DOC_STRING(
+      "Gets @IReactPropertyBag to share values between components and the application."
+      "Use @IReactContext.Properties to access this @IReactPropertyBag from native components and view managers.")
     IReactPropertyBag Properties { get; };
 
-    DOC_STRING("Provides access to the `IReactNotificationService`, which allows easy communication between custom native modules or view managers.")
+    DOC_STRING(
+      "Gets @IReactNotificationService to send notifications between components and the application.\n"
+      "Use @IReactContext.Notifications to access this @IReactNotificationService "
+      "from native components or view managers.")
     IReactNotificationService Notifications { get; };
 
-    DOC_STRING("Provides a list of additional native modules and custom view managers that should be included in the instance.")
+    DOC_STRING(
+      "Gets a list of @IReactPackageProvider.\n"
+      "Add an implementation of @IReactPackageProvider to this list to define additional "
+      "native modules and custom view managers to be included in the React instance.\n"
+      "Auto-linking automatically adds @IReactPackageProvider to the application's @.PackageProviders.")
     IVector<IReactPackageProvider> PackageProviders { get; };
 
-    DOC_STRING("This controls whether various developer experience features are available for this instance.  In particular the developer menu, the default `RedBox` and `LogBox` experience and loading UI during bundle load.")
+    DOC_STRING(
+      "This controls whether various developer experience features are available for this instance. "
+      "In particular, it enables the developer menu, the default `RedBox` and `LogBox` experience, "
+      "and loading UI during the JavaScript bundle load.")
     Boolean UseDeveloperSupport { get; set; };
 
     // Deprecated - use JavaScriptBundleFile instead
-    DOC_STRING("Name of the JavaScript bundle file.  If @.JavaScriptBundleFile is specified it is used instead.")
-    [deprecated("use JavaScriptBundleFile instead", deprecate, 1)]
+    [deprecated("Use JavaScriptBundleFile instead. It will be removed in version 0.65.", deprecate, 1)]
```

Comment:
```suggestion
    [deprecated("Use @.JavaScriptBundleFile instead. It will be removed in version 0.65.", deprecate, 1)]
```

Reply:
This is text in the deprecated attribute. It does not use the same rules for references as doc comments.